machine_learning/sequential_minimum_optimization.py

```diff
@@ -28,7 +28,6 @@
 Reference:
     https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/smo-book.pdf
     https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/tr-98-14.pdf
-    https://web.cs.iastate.edu/~honavar/smo-svm.pdf

```

Comment:
This link does not exist; hence I removed it.

Reply:
Please create a new issue asking the author of this file to suggest a replacement.